tests/scripts/check-files.py

```diff
@@ -290,7 +290,8 @@ def setup_logger(self, log_file, level=logging.INFO):
             console = logging.StreamHandler()
             self.logger.addHandler(console)
 
-    def collect_files(self):
+    @staticmethod
```

Comment:
The commit message looks off. Did you mean to squash this commit into the previous one?

Reply:
Yes.